app/views/handlers/view_submitted_response_pdf.py

```diff
@@ -54,7 +54,11 @@ def __init__(
     @property
     def filename(self) -> str:
         """The name to use for the PDF file"""
-        return f"{self._metadata['schema_name']}.pdf"
+        formatted_title = (
+            self._schema.json["title"].lower().replace(" ", "-").replace("--", "-")
+        )
```

Comment:
Currently this will still result in `--` in the title. Thinking about it, worth just replacing all alphanumeric chars would something like:
```py
formatted_title = re.sub('[^0-9a-zA-Z]+', '-', self._schema.json["title"].lower())
```
This fixes the issue above + handles other non alphanum chars. Used regex on purpose over built-in `isalnum` as it won't result in same behaviour. We want to replace non alpha num instead of not outputting it.

Reply:
Ok yeah good idea Ive added it in